teams/no-team.yml

```diff
@@ -0,0 +1,10 @@
+# Teams are available in Fleet Premium. 
+
+# This file updates policies, controls, and software for hosts assigned to "No team."
```

Comment:
@lucasmrod is it true that we only support policies, controls, and software for "No team"? (no queries or agent options yet)

And, queries and agent options for hosts on "No team" are controlled at the global (default.yml) level.

Reply:
- Queries: We don't support queries for "No team", so, queries defined in `default.yml` are global queries.
- Agent options: Agent options for "No team" hosts are defined in `default.yml` (in the future we could move this to `teams/no-team.yml`). On this iteration Marko adviced (in the Figma designs) to throw a warning if the user is setting `agent_options` or `queries` in `teams/no-team.yml` (they will be ignored).

Reply:
Got it! Thanks :)

I updated the help text (comments at the top of each file) to be explicit. Please let me know if those doesn't look right.